bak2dvd/bak2dvd

```diff
@@ -0,0 +1,268 @@
+#!/usr/bin/env bash
+# Author: Charles Oblender https://github.com/oblende
+currentVersion="1.0"
+configuredClient=""
+
+dvdburn() {
+	counter=$4
+	while [ "$counter" -eq 1 ]
+	do
+        	growisofs -Z "$1"=$2
+        	counter=0
+        	counter=$(ps -A | grep $3 | wc -l)
+        	if [[ $counter -eq 1 ]]
+        	then
+                	read -p "Insert next disc and press Enter:"
+                	echo >&7
+        	fi
+	done
+}
+
+cdburn() {
+	counter=$4
+	while [ "$counter" -eq 1 ]
+	do
+		wodim dev=$1 -tao -multi -eject -data $2
```

Comment:
Doesn't seem like ```wodim``` is available on all of the desired machines, are there any cross platform alternatives?

Reply:
Cdrecord works with FreeBSD, and likely others. Will need unames of other systems. Might possibly only need a default in the case statement and an assumption that `ps -A` works like the BSD version of it.

Reply:
Found out about all the command line opensource CD burners that could be included in any OS distribution. Script should be able to fine one and work when backing up to CD. This should satisfy cross platform needs.